### PR TITLE
IR intensities

### PR DESCRIPTION
**Pull request by dsirianni**
## Description
Will add reference implementation(s) and tutorial(?) for computing IR intensities of molecules.

## What are your new additions? Please provide a brief list.
* **New Features**
  - [x] Pure-Python implementation from CCQC reference files (in `ccqc/` subdir)
  - [x] Psi4NumPy reference implementation using Andy's dipole derivatives

## Any questions for the community?
- [ ] Worth keeping initial implementation based on CCQC project after dipole derivatives verified?

## Status
- [x] Ready for review
- [ ] Ready for merge

### CODE REVIEW

**Comment by dgasmith** [comment]
Cool! Looking forward to this.

I do not think we would like the current development files in the master branch. It is good to have them for now, but best to remove them from the history once we have the new code in place.

**Comment by loriab** [comment]
This depends on psi4/psi4#1382, so it'll never pass CI right now. And that PR will need to be merged to get a psi4 version tag to tell the CI to conditionall run.

File: One-Electron-Property/IR-Intensities/intensities.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                 })
+
+masses = np.array([mol.mass(i) for i in range(mol.natom())])
+
+# Get Hessian from wavefunction
+e, wfn = psi4.frequencies('scf', return_wfn=True)
+Hess = np.asarray(wfn.hessian())
+
+# Read in dipole derivatives
+dipdir = wfn.dipole_gradient().np.T
```
[the remark this answers is not in the document]
Comment (andysim) [reply One-Electron-Property/IR-Intensities/intensities.py]: Your solution is correct though.  We'll be adding correlated dipole gradients from finite differences in future, so IR intensities will be available for all methods.

File: One-Electron-Property/IR-Intensities/ccqc-intensities.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+# Compute IR intensities according to [CCQC:Project2] Eqn. 16
+I = np.zeros(dmdQ.shape[1])
+
+print(" ==> IR Intensities of Ethylene @ HF/sto-3g <== ")
+print(" Mode     (D^2 / AA^2 amu) \t a.u. \t\tkm/mol ")
+print("-------------------------------------------------------")
+for i in range(dmdQ.shape[1]):
+    I[i] = dmdQ[:,i].dot(dmdQ[:,i]) # Native units in D^2 / AA^2 amu
+    print('  %s   \t\t%6.4f  \t%6.4f  \t%6.4f' % (i, I[i], I[i] * bohr2angstrom**2, natI2kmmol*I[i]))
```
Comment (dgasmith) [inline One-Electron-Property/IR-Intensities/ccqc-intensities.py]: I think if we remove this file and the `ccqc/*` we should be good to go.

Comment (dgasmith) [reply One-Electron-Property/IR-Intensities/ccqc-intensities.py:85]: @dsirianni It looks like this is one tweak away from mergeable.

**Comment by dgasmith** [comment]
@dsirianni FYI, I think @kcpearce is going to take crack at finishing this up.

**Comment by loriab** [comment]
feel free to take anything you like from vib.py in psi4

**Comment by kcpearce** [comment]
Can probably close this now that #104 has been merged

**Comment by dgasmith** [comment]
+1 thanks.

**Pull request closed without merging**